src/django_peertube_runner_connector/utils/thumbnail.py

```diff
@@ -0,0 +1,32 @@
+"""Thumbnail utilities."""
+import tempfile
+
+import ffmpeg
+
+from django_peertube_runner_connector.models import Video, VideoFile
+from django_peertube_runner_connector.storage import video_storage
+
+from .files import get_video_directory
+
+
+def build_video_thumbnails(video=Video, video_file=VideoFile):
+    """Create a video thumbnails with ffmpeg and save it to a file."""
+    video_url = video_storage.url(video_file.filename)
+    thumbnail_filename = get_video_directory(video, "thumbnail.jpg")
+
+    with tempfile.NamedTemporaryFile(suffix=".jpg") as temp_file:
+        output_path = temp_file.name
+
+        input_stream = ffmpeg.input(video_url)
+
+        output_stream = ffmpeg.filter(
+            input_stream,
+            "select",
+            f"gte(n,{video.duration / 2})",
```

Comment:
Finally you choose to take a frame in the middle of the video ?

Reply:
nop I just forgot to change it